src/mainboard/clevo/adl-p/Kconfig

```diff
@@ -0,0 +1,68 @@
+if BOARD_SYSTEM76_NS70PU
+
+config BOARD_SPECIFIC_OPTIONS
+	def_bool y
+	select BOARD_ROMSIZE_KB_32768
+	select DRIVERS_I2C_HID
+	select DRIVERS_INTEL_PMC
+	select DRIVERS_INTEL_USB4_RETIMER
+	select EC_SYSTEM76_EC
+	select EC_SYSTEM76_EC_COLOR_KEYBOARD if BOARD_SYSTEM76_NS70PU
+	select HAVE_ACPI_TABLES
+	select HAVE_CMOS_DEFAULT
+	select HAVE_OPTION_TABLE
+	select HAVE_SPD_IN_CBFS if BOARD_SYSTEM76_LEMP11
+	select INTEL_GMA_HAVE_VBT
+	select INTEL_LPSS_UART_FOR_CONSOLE
+	select MAINBOARD_HAS_TPM2
+	select MEMORY_MAPPED_TPM
+	select NO_UART_ON_SUPERIO
+	select SOC_INTEL_ALDERLAKE_PCH_P
+	select SOC_INTEL_COMMON_BLOCK_HDA_VERB
+	select SOC_INTEL_CRASHLOG
+	select SOC_INTEL_ENABLE_USB4_PCIE_RESOURCES
+	select SPD_READ_BY_WORD
+	select SYSTEM_TYPE_LAPTOP
+	select TPM_MEASURED_BOOT
+
+config MAINBOARD_DIR
+	default "clevo/adl-p"
+
+config VARIANT_DIR
+	default "ns70pu" if BOARD_SYSTEM76_NS70PU
+
+config OVERRIDE_DEVICETREE
+	default "variants/\$(CONFIG_VARIANT_DIR)/overridetree.cb"
+
+config MAINBOARD_PART_NUMBER
+	default "ns70pu" if BOARD_SYSTEM76_NS70PU
+
+config MAINBOARD_SMBIOS_PRODUCT_NAME
+	default "Darter Pro" if BOARD_SYSTEM76_NS70PU
```

Comment:
Change according to the actual SMBIOS dump from the laptop. We want these SMBIOS fields to match vendor firmware since they affect automatic driver installation from Windows Update

Reply:
Fixed.